imgui-examples/examples/creating_windows.rs

```diff
@@ -0,0 +1,67 @@
+mod support;
```

Comment:
should we show off using `imgui::Window::new`? I think probably no, yeah?

Reply:
Might as well mention it as long as its available, even if just in a "probably don't use this" way